templates/base.html

```diff
@@ -57,14 +82,14 @@
               <a class="nav-link" href="/tips">Tips</a>
             </li>
           </ul>
-            {% if user.is_authenticated %}
-                <a id class="nav-link mr-sm-2" href="/accounts/logout">Logout</a>
-            {% else %}
-                <a id class="nav-link mr-sm-2" href="/accounts/login">Login</a>
-            {% endif %}
+          {% if user.is_authenticated %}
+          <a id class="nav-link mr-sm-2" href="/accounts/logout">Logout</a>
+          {% else %}
+          <a id class="nav-link mr-sm-2" href="/accounts/login">Login</a>
```

Comment:
I see that this button has moved to the middle
Did you mean that this will happen? I think it will be nicer in the top right corner

Reply:
I agree with Rony, I think it could be better on the right side

Reply:
I did not meant for this to happen, I created the PR before this was merged and this probably happend with the rebase